### ISSUE TITLE
2022.01.07-1.77.2-nb Settings changes couldn't be saved.

### ISSUE THREAD

**Issue by Mordrag** [mRemoteNG/mRemoteNG#2128]
I Updated my mremote to the new Nightly build. Now i got Problems in saving any changes in the options or the layout. And if you activate or deactivate some Symbol bars you have to restart mremote, otherwise they didn't appear or disappear.

Is this a known bug?

I Use Windows 11 with all actuall installes Hotfixes from 01.22.

**Comment by Kvarkas** [comment]
Hi, i am on win11 as well, could you describe a bit more that exactly is not saving?

**Comment by Mordrag** [comment]
if i change "save connections on exit" in options, click Okay and reopen the Optionsmenu, the option is unchecked again. The other problem is if you activate any symbol bar in the menü, you have to restart the mRemoteNG Client to get the bar.
Than you can place the bars how you want, after a client restart you have to sort them again. At the moment i activated the "external apps" bar.
The position of this bar can't be saved also on my system.

**Comment by david-sway** [comment]
I am also observing that the "save connections on exit" setting is not persisted.

The other issue with enabling the toolbars was fixed in https://github.com/mRemoteNG/mRemoteNG/pull/2114, which your 2022.01.07 build predates

**Comment by Kvarkas** [comment]
will release new nb tonight so we may have new changes in for more tests.

**Comment by andipohl** [comment]
Is there any progress with this problem in the meantime?

**Comment by Kvarkas** [comment]
hello working on that part, its a bit messy :) but hope will finish this weekend

**Comment by Mordrag** [comment]
is this fixed in the new nightly? I updated to it, but my external apps bar always moved to the second line if i restart mremote and now i am not able to get the connect and config panel both on the left side.

**Comment by simonai1254** [comment]
In 1.77.3-nb the Settings Saving is completely disabled for development purposes, so even if they would have implemented the fix for the option itself it would help as the complete saving process is bypassed...

**Comment by Kvarkas** [comment]
@Mordrag its fixed in last nb release, please check: https://github.com/mRemoteNG/mRemoteNG/releases/download/2023.03.03-v1.77.3-nb/mRemoteNG-Installer-1.77.3.nb-1784.msi

**Comment by Kvarkas** [comment]
yes it will not remember that but in last nb it allow panels work properly (due library update)

**Comment by Mordrag** [comment]
i installed the Nightly v1.77.3.1442-NB. There it should work? Because it doesn't.

**Comment by Mordrag** [comment]
Ok i installed the Version 1784, but also didn't work. the External Apps panel switch always to the second line!

**Comment by GotSQL** [comment]
For confirmation, I've installed "mRemoteNG-Installer-1.77.3.nb-1784.msi," and when changing settings, they do not save. On a side note, I usually stick to stable, but I've read this build resolves the log4j issue with the specific dll file used in other versions. I also understand that saving the settings on a NB is not typical, but I thought I saw in the above comments that there was an exception made ( https://github.com/mRemoteNG/mRemoteNG/issues/2128#issuecomment-1488201072 ). Thank you for the work on this project.

SPECIFICS NOT SAVING
- moving the Quick Connect toolbar up a row, next to the Help so it is all on one row; on reopen, that change is gone
- Opening a new config file that is stored outside of the default folder structure of mRemoteNG, mRemoteNG on close and reopen does not remember or open the last config file used.

Using latest Windows 11 22H2 fully updated.

**Comment by Kvarkas** [comment]
not relevant
